pkg/curatedpackages/packagecontrollerclient_test.go

```diff
@@ -54,14 +75,52 @@ func TestInstallControllerSuccess(t *testing.T) {
 	registry := curatedpackages.GetRegistry(tt.ociUri)
 	sourceRegistry := fmt.Sprintf("sourceRegistry=%s", registry)
 	values := []string{sourceRegistry}
+	params := []string{"create", "-f", "-", "--kubeconfig", tt.kubeConfig}
+	templateValues := map[string]string{
+		"eksaAccessKeyId":     "",
+		"eksaSecretAccessKey": "",
+		"eksaRegion":          base64.StdEncoding.EncodeToString([]byte(eksaDefaultRegion)),
+	}
+	result, err := templater.Execute(awsSecretYaml, templateValues)
+	tt.Expect(err).To(BeNil())
```

Comment:
nit: I tend to use this, a bit more expressive imo but not big deal
```suggestion
	tt.Expect(err).NotTo(HaveOcurred())
```